io.github.plrigaux.sysd-manager.yaml

```diff
@@ -0,0 +1,64 @@
+id: io.github.plrigaux.sysd-manager
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+#runtime: org.freedesktop.Platform
+#runtime-version: '23.08'
+#sdk: org.freedesktop.Sdk
+sdk-extensions:
+- org.freedesktop.Sdk.Extension.rust-stable
+- org.freedesktop.Sdk.Extension.llvm16
+command: sysd-manager
+finish-args:
+- --share=ipc
+- --socket=fallback-x11
+- --socket=wayland
+- --device=dri
+#- --filesystem=home
+- --filesystem=host
+#- --talk-name=org.freedesktop.Flatpak"
+- --env=RUST_LOG=info
+- --env=G_MESSAGES_DEBUG=none
+- --env=RUST_BACKTRACE=1
+- --env=SYSDMNG_DIST_MODE=flatpack
+# Typical GNOME and GTK applications should use:
+- --talk-name=org.gtk.vfs.*
+- --filesystem=xdg-run/gvfsd
+# Needed to talk to systemd1 with dbus
+- --talk-name=org.freedesktop.systemd1
+- --system-talk-name=org.freedesktop.systemd1
+
```

Comment:
why do you need this ?

Reply:
The program is based on having dbus access to systemd. It does api calls as defined here https://www.freedesktop.org/wiki/Software/systemd/dbus/ to retrieve the list of units (services), start service, stop service, get unit parameters, ...

Basically the program is a GUI of "systemctl". Without those 2 access the program is meaningless.

Thank you